tests/readme.md

```diff
@@ -100,11 +100,11 @@ mvn verify -Prun-tests -Dhono.deviceregistry.type=file
 The tests can be run in such a way, that the OpenTracing trace spans created in the Hono components
 as part of a test run can be inspected later on. The OpenTracing component used for this is Jaeger.
 
-To include the Jaeger client, build the Hono Docker images using the `jaeger` Maven profile:
+To include the Jaeger client, build the Hono Docker images using the `opentelemetry` Maven profile:
```

Comment:
This does not really include the Jaeger client, does it?

Reply:
I've adapted the documentation now. I've kept the `jaeger` profile in the `tests` pom as this still involves using Jaeger as tracing backend and UI.